src/utils/device.js

```diff
@@ -96,3 +96,30 @@ module.exports.isLandscape = function () {
 module.exports.isIOSOlderThan10 = function (userAgent) {
   return /(iphone|ipod|ipad).*os.(7|8|9)/i.test(userAgent);
 };
+
+/**
+ * Check if running in a browser.
+ */
+// we need to check a node api that isn't mocked on either side
```

Comment:
I'm not sure if I did the commenting right. I was trying to separate the public comments that go in the api docs, from private implementation comments.